src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -4054,53 +4080,31 @@ public String getDownloadType() {
     public void setDownloadType(String downloadType) {
         this.downloadType = downloadType;
     }
-    
-    
+
+
     public void openDownloadPopupForMultipleFileDownload() {
         if (this.selectedFiles.isEmpty()) {
             //RequestContext requestContext = RequestContext.getCurrentInstance();
             PrimeFaces.current().executeScript("PF('selectFilesForDownload').show()");
```

Comment:
should we consider removing this one while we're here as well? (or is that out of scope)

Reply:
Yeah, it's only used for the download mixed popup (some restricted; some not) so we know there are valid files selected so we can just put the openguestbook popup in the onComplete.